<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newAct"/>
    <addaction name="newWAct"/>
    <addaction name="openAct"/>
    <addaction name="saveAct"/>
    <addaction name="saveSAct"/>
    <addaction name="priAct"/>
    <addaction name="closeAct"/>
    <addaction name="exitAct"/>
    <addaction name="closeWAct"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="revokeAct"/>
    <addaction name="cutAct"/>
    <addaction name="copyAct"/>
    <addaction name="patseAct"/>
    <addaction name="delAct"/>
    <addaction name="selectAllAct"/>
    <addaction name="dateAct"/>
   </widget>
   <widget class="QMenu" name="menuc">
    <property name="title">
     <string>查看</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>缩放</string>
     </property>
     <addaction name="enlargeAct"/>
     <addaction name="reduceAct"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="statusBarAct"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menuc"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newAct">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>新建标签页</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="newWAct">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>新建窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="openAct">
   <property name="icon">
    <iconset resource="applicationIcon.qrc">
     <normaloff>:/Icon/openfile.png</normaloff>:/Icon/openfile.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="icon">
    <iconset resource="applicationIcon.qrc">
     <normaloff>:/Icon/savefile.png</normaloff>:/Icon/savefile.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveSAct">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="priAct">
   <property name="text">
    <string>打印</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="closeAct">
   <property name="text">
    <string>关闭选项卡</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="revokeAct">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="cutAct">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="icon">
    <iconset resource="applicationIcon.qrc">
     <normaloff>:/Icon/copy.png</normaloff>:/Icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="patseAct">
   <property name="icon">
    <iconset resource="applicationIcon.qrc">
     <normaloff>:/Icon/patse.png</normaloff>:/Icon/patse.png</iconset>
   </property>
   <property name="text">
    <string>黏贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="delAct">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="selectAllAct">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="dateAct">
   <property name="text">
    <string>时间/日期</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="statusBarAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>状态栏</string>
   </property>
  </action>
  <action name="closeWAct">
   <property name="text">
    <string>关闭窗口</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="enlargeAct">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="reduceAct">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="applicationIcon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exitAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeWAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
